front/src/common/redux/groups/groups.api.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+
+async function updateGroup(request){
+    const { groupName, id } = request.data;
+    const token = localStorage.getItem('token');
+    const url = `${process.env.REACT_APP_GROUPS_LOAD_AND_UPDATE}${id}`;
+    const data = { groupName };
+    if (request.data.avatar.img) {
+        data.avatar = request.data.avatar;
+    }
+    const headers = {
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${token}`,
+        }
+    };
+    return axios.put(url, data, headers)
+        .then(response => response)
```

Comment:
Please, return only data from API function, don't return the full response, saga should not know anything about HTTP requests

Reply:
done